### PR TITLE
Feature/hw 6

### PR DESCRIPTION
**Pull request by LevapVeeskela**

### CODE REVIEW

File: ItAcademyHomeWorks/BusinessLogic/Services/HW_06/FirstTaskService.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    public class FirstTaskService
+    {
+        public void GetVerse()
+        {
+            string[] verseArray;
+
+            if (WriteOrReadConsoleHelper.GetPromptConfirmation(HomeWorkSix.TextConfirmInfo))
+            {
+                var pathVerseBlank = Path.Combine(HomeWorkSix.PathSolution, HomeWorkSix.NameFile);
+                var fullPathFile = WriteOrReadConsoleHelper.GetPromptConfirmation(pathVerseBlank)
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/BusinessLogic/Services/HW_06/FirstTaskService.cs]: usually it looks like below

Comment (VadzimPapko) [reply ItAcademyHomeWorks/BusinessLogic/Services/HW_06/FirstTaskService.cs]: image

File: ItAcademyHomeWorks/BusinessLogic/Services/HW_06/FirstTaskService.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        {
+            return ReplaceLetter(File.ReadAllText(file.FullName, Encoding.UTF8), HomeWorkSix.DefaultOriginChangeLetter, HomeWorkSix.DefaultChangeLetter).Split(separator);
+        }
+
+        private string ReplaceLetter(string text, char originLetter, char changeLetter)
+        {
+            return text.Replace(originLetter, changeLetter);
+        }
+    }
+}
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/BusinessLogic/Services/HW_06/FirstTaskService.cs]: well done

File: ItAcademyHomeWorks/Common/Extensions/StringExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                MaxWord = arrayRow.FirstOrDefault(),
+                MinWord = arrayRow.LastOrDefault()
+            };
+        }
+
+        /// <summary>
+        /// Get count letter in row only
+        /// </summary>
+        /// <param name="row">Origin row</param>
+        /// <returns></returns>
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/Common/Extensions/StringExtensions.cs]: returns element is empty

Comment (VadzimPapko) [reply ItAcademyHomeWorks/Common/Extensions/StringExtensions.cs]: just keep in mind - each expression in documentation code should contain Dot character `.` at the end

File: ItAcademyHomeWorks/Common/Models/RowInfoModel.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
+﻿namespace Common.Models
+{
+    public class RowInfoModel
+    {
+        /// <summary>
+        /// OriginRow
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/Common/Models/RowInfoModel.cs]: dot character

File: ItAcademyHomeWorks/Common/Extensions/StringExtensions.cs (hunk 3 above)
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/Common/Extensions/StringExtensions.cs]: ok, I don't know it, thanks)

File: ItAcademyHomeWorks/BusinessLogic/Services/HW_06/FirstTaskService.cs (hunk 1 above)
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/BusinessLogic/Services/HW_06/FirstTaskService.cs]: I agree, I fix next time

File: ItAcademyHomeWorks/Common/Models/RowInfoModel.cs (hunk 4 above)
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/Common/Models/RowInfoModel.cs]: got it

**Pull request merged**